### PR TITLE
koord-scheduler: loadAwareScheduling skip node without NodeMetric in …

### LINKED ISSUE #316: [BUG] koord-scheduler failed to schedule Pod without NodeMetric not found
**What happened**:

Some nodes in the cluster are used for co-location. When creating Pod scheduling, it is required to schedule to non-co-location nodes. At this time, Pod cannot be scheduled, and the related error message is nodemetrics not found.

**What you expected to happen**:

koord-scheduler scheduled  the new created Pod  successfully even if there is not NodeMetrics in the node.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- App version:
- Kubernetes version (use `kubectl version`):
- Install details (e.g. helm install args):
- Others:

### PR DESCRIPTION
**Pull request by eahydra**
…filter/score phase

Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### Ⅱ. Does this pull request fix one issue?

fix #316 

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/scheduler/plugins/loadaware/load_aware.go [hunk 1]
```diff
@@ -102,6 +103,9 @@ func (p *Plugin) Filter(ctx context.Context, state *framework.CycleState, pod *c
 
 	nodeMetric, err := p.nodeMetricLister.Get(node.Name)
 	if err != nil {
+		if errors.IsNotFound(err) {
```
Comment (hormes) [inline pkg/scheduler/plugins/loadaware/load_aware.go:106]: please add some comments like:

For nodes that lack load information, fall back to the situation where there is no load-aware scheduling, and deal with the scenario where the user installs the koordlet component according to the grayscale of the nodes in the cluster.

Comment (eahydra) [reply pkg/scheduler/plugins/loadaware/load_aware.go:109]: done

**Pull request merged**